fmc_rest_client/resources/objects.py

```diff
@@ -91,3 +91,9 @@ def __init__(self, name=None, tag=None, desc=None):
         super().__init__(name)
         self.tag = tag
         self.description = desc
+
+class VariableSet(ObjectResource):
```

Comment:
The json_load will load only the attributes defined here from a json passed to it, do you just need name and desc here?